src/interfaces/ffi/hybrid_cc_aes.rs

```diff
@@ -911,8 +913,8 @@ pub unsafe extern "C" fn h_aes_encrypt(
 ///
 /// # Safety
 pub unsafe extern "C" fn h_aes_decrypt(
-    plaintext_ptr: *mut c_char,
-    plaintext_len: *mut c_int,
+    cleartext_ptr: *mut c_char,
+    cleartext_len: *mut c_int,
```

Comment:
@bgrieder said we should use "plaintext" instead of "cleartext", no?

Reply:
right